README.md

```diff
@@ -2,24 +2,214 @@
+
+As with the process for Provisional acceptance, the "**Straw Poll**" mechanism is utilized to provide an initial screening of the proposal. Passing of the straw poll _does not_ confer elevation to Stable status, but serves instead as a means for quickly pushing back proposals that lack significant support. This is intended to both protect the time of ASC members and author(s), and to help guide author(s) wishing to continue pursuit of the proposal.
+
+Upon successfully completing the straw poll, the Co-Chairs shall label the PR as "**Eligible**" to indicate that the proposal is eligible for formal consideration by the ASC. Given the significant commitment conferred by the PMIx community to "Stable" elements, formal approval of elevation requires that the ASC conduct a full presentation on the proposal at two consecutive quarterly meetings. At the conclusion of the first meeting, a formal vote of the ASC shall be held (per the above voting rules) to determine if the proposal merits further consideration. The Co-Chairs shall set a GitHub label indicating the result as either "**First Pass**" or "**Pushed Back**".
+
+At the conclusion of the second meeting where the proposal is up for consideration, another formal vote of the ASC shall be held (per the above voting rules) to determine final disposition of the proposal. The Co-Chairs shall subsequently set a GitHub label indicating the result as either "**Stable**" (indicating that the proposed elements have been elevated to that class) or "**Pushed Back**".
+
+Proposals that are pushed back at any point can be modified and resubmitted by the author(s). However, the updated PR must pass through the complete elevation process from the beginning to ensure full consideration of the revised request. A new PR for this purpose _must not_ be submitted for the identical elements in order to retain full information regarding prior concerns and suggestions.
+
+
+#### Deprecation
+Deprecation of one or more existing elements of the Standard, be they APIs or attributes, starts by mirroring the path of a newly proposed modification. An Issue proposing the deprecation and explaining the reasons behind it shall be opened for initial discussion. If desired, this may immediately be accompanied by the associated PR to remove the identified element(s), or the PR can be generated after an appropriate amount of discussion has concluded. The deprecation PR must include the required "**DEPRECATION**" warning language to be added to the Standard - examples are provided in the GitHub Deprecation Template. The same "**Straw Poll**" voting mechanism, including presentation at a quarterly meeting, shall be used to determine if the proposal merits deeper consideration by the ASC.
```

Comment:
> "this may immediately be accompanied by the associated PR to remove the identified element(s)"

I think we first need a PR to mark the API as deprecated that the RMs can commit right away. Then a second PR which will remove it. This allows us to release a standard A.0 with the API labeled as deprecated but still included in the document. Then at C.0 it can be formally removed.

Maybe the timeline can be:
 * A.0 : API/Attribute marked as deprecated in place in the document. Update change listing to indicate it being added as newly deprecated. PR specific to this change by the authors.
 * B.0 : API/Attribute marked as deprecated in place in the document. Update change listing to indicate possible removal in next major version. PR specific to this change by the RMs (since it's a changelog item).
 * C.0 : API/Attribute removed from the document. Update change listing to indicate removal from the standard. PR specific to this change by the authors & RMs.

Reply:
Not sure I understand your comment - the paragraph states that a PR is required. It can either be generated immediately upon opening the Issue, or after some period of discussion. Is that not clear?

Or is your comment not so much about the PR but more about wanting two separate PRs - one to deprecate and one to remove? I guess I can see that so the PR can contain the actual change.

Reply:
Here is what I modified in the deprecation section:

> If desired, this may immediately be accompanied by the associated PR to mark the identified element(s) as deprecated, or the PR can be generated after an appropriate amount of discussion has concluded.

...

> At the conclusion of the deprecation cycle, a second "removal" PR shall be submitted to actually remove the deprecated element(s) from the PMIx Standard document. When ready for consideration, the author(s) shall label the PR with the "Removal" tag, thereby requesting that the matter be placed on the agenda of the next quarterly meeting (subject to the one month deadline). The "straw poll" approval process utilized for "Provisional" proposals is utilized to ensure adequate review is performed to avoid inadvertent removal of elements not included in the original deprecation proposal. Once the process has been concluded, the Release Manager shall commit the change in accordance with their schedule.

Reply:
Yeah, 1 PR to depreciate, and 1 PR to remove. The wording that was there was that you file 1 PR to remove meaning that the RMs have to figure out all of the places in the document to place a deprecation warning. The depreciation PR also forces the authors to identify all of the places where that API/Attribute is used and add some warning about depreciation in the future. This can help the reviewers understand the full extent of the deprecations impact.

What you have is good. I would suggest changing the first block to make it a bit more clear that a PR is required for deprecation before voting. The wording you have makes the PR optional. Maybe something like:
> The authors must file a Depreciation PR for voting which identifies the specific changes to the document where deprecation warnings must be added and/or wording needs to be modified. Discussion may occur on the Issue before a PR is generated, but a PR is necessary for voting.